src/components/APIPublications/APIPublications.tsx

```diff
@@ -190,9 +176,10 @@ export const APIPublications: React.FC<APIPublicationsProps> = ({
           <Grid item md={3}>
             <Box mt={4} style={{ textAlign: "right" }}>
               <Button
+                disabled={!apiDetails.version!.specFile}
```

Comment:
```suggestion
                disabled={!apiDetails.version?.specFile}
```
since there's the possibility for specFile to be null